package-lock.json

```diff
@@ -10967,7 +10967,8 @@
         },
         "yargs-parser": {
           "version": "13.1.1",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-13.1.1.tgz",
```

Comment:
Not sure if I should preserve these updates or strip.

Reply:
Right, npm seems to vacillate between resolving and not resolving this dependency. Not sure why, but we probably shouldn’t change package-lock in this PR.